### PR TITLE
fixes spelling errors

### PR DESCRIPTION
**Pull request by rose2221**
##### SUMMARY

I've addressed the spelling errors in the files specified.

Solves #81808

### CODE REVIEW

**Comment by rose2221** [comment]
there are many incorrect paths for the files sepcified in issue #81808

**Comment by rose2221** [comment]
However, several files either don't exist or have incorrect paths, preventing me from rectifying their spelling issues. here is the list of the paths that doesn't exist:-
lib/ansible/modules/extract_filter.py:221: contianer ==> container
lib/ansible/modules/from_yaml_all_filter.py:150: mulitple ==> multiple
lib/ansible/modules/mandatory_filter.py:37: existance ==> existence
lib/ansible/modules/pow_filter.py:66: inputed ==> inputted
lib/ansible/modules/realpath_filter.py:67: alwasy ==> always
lib/ansible/modules/root_filter.py:66: inputed ==> inputted
lib/ansible/modules/strftime_filter.py:37: formating ==> formatting
lib/ansible/modules/strftime_filter.py:66: formating ==> formatting
lib/ansible/modules/strftime_filter.py:66: formated ==> formatted
lib/ansible/modules/strftime_filter.py:125: formating ==> formatting
lib/ansible/modules/to_uuid_filter.py:128: fo ==> of
lib/ansible/modules/dict_lookup.py:227: dictonaries ==> dictionaries
lib/ansible/modules/changed_test.py:67: existance ==> existence
lib/ansible/modules/failed_test.py:67: existance ==> existence
lib/ansible/modules/finished_test.py:66: aslo ==> also
lib/ansible/modules/finished_test.py:67: existance ==> existence
lib/ansible/modules/finished_test.py:206: aysnc ==> async
lib/ansible/modules/match_test.py:183: insenstive ==> insensitive
lib/ansible/modules/match_test.py:228: mulitple ==> multiple
lib/ansible/modules/reachable_test.py:67: existance ==> existence
lib/ansible/modules/regex_test.py:183: insenstive ==> insensitive
lib/ansible/modules/search_test.py:184: insenstive ==> insensitive
lib/ansible/modules/search_test.py:229: mulitple ==> multiple
lib/ansible/modules/skipped_test.py:67: existance ==> existence
lib/ansible/modules/started_test.py:67: existance ==> existence
lib/ansible/modules/success_test.py:67: existance ==> existence
lib/ansible/modules/unreachable_test.py:67: existance ==> existence
please look into it @samccann

File: lib/ansible/modules/get_url.py [hunk 1]
```diff
@@ -261,7 +261,7 @@
 
 - name: Download file from a file path
   ansible.builtin.get_url:
-    url: file:///tmp/afile.txt
+    url: file:///tmp/a_file.txt
```
Comment (bcoca) [inline lib/ansible/modules/get_url.py:264]: not really a spelling error, but not against the change

File: lib/ansible/modules/yum.py [hunk 2]
```diff
@@ -241,7 +241,7 @@
         registered to RHN or an RHN Satellite, repoquery allows for querying
         all channels assigned to the system. It is also required to use the
         'list' parameter.
-      - "NOTE: This will run and be logged as a separate yum transation which
+      - "NOTE: This will run and be logged as a separate yum transaction, translation which
```
Comment (bcoca) [inline lib/ansible/modules/yum.py]: remove `, translation`

**Review by bcoca — CHANGES_REQUESTED** [review]
just some extra text to remove, rest lgtm

**Comment by bcoca** [comment]
@rose2221 the list of 'missing files' are just incorrectly located and named, they seem to follow 'module rules for docs' but they are filters and tests. Filters live in `lib/ansible/plugins/filter` and tests live in `lib/ansible/plugins/test` and you can remove the `_test.py` and `_filter.py` in exchange for a `.yml`.

File: lib/ansible/modules/yum.py (hunk 2 above)
Comment (rose2221) [reply lib/ansible/modules/yum.py]: done

Comment (rose2221) [reply lib/ansible/modules/yum.py]: i have done the required changes, please look into it @bcoca

**Review by bcoca — APPROVED** [review]

**Pull request merged**